src/supremm/ingest_jobscripts.py

```diff
@@ -208,6 +219,23 @@ def getoptions():
     return retdata
 
 
+def parse_resource_config(settings):
+    """ Parse the resource configuration settings. Support both the 1.3.0 and
+        1.4.0 configuration styles """
+
+    respath = None
+    timestamp_mode = 'start'
+
+    if 'batchscript' in settings:
+        if 'path' in settings['batchscript']:
+            respath = settings['batchscript']['path']
+        if 'timestamp_mode' in settings['batchscript']:
+            timestamp_mode = settings['batchscript']['timestamp_mode']
```

Comment:
I think it would be good to validate the `timestamp_mode` to prevent issues with typos. I don't see that happening anywhere.